packages/launchdarkly-adapter/modules/adapter/adapter.ts

```diff
@@ -234,27 +229,29 @@ const getInitialFlags = ({
   );
 };
 
-const configure = ({
-  clientSideId,
-  user,
-  clientOptions = {},
-  onFlagsStateChange,
-  onStatusStateChange,
-  flags,
-  subscribeToFlagChanges = true,
-  throwOnInitializationFailure = false,
-  flagsUpdateDelayMs,
-}: {
-  clientSideId: string;
-  user: User;
-  clientOptions: ClientOptions;
-  onFlagsStateChange: OnFlagsStateChangeCallback;
-  onStatusStateChange: OnStatusStateChangeCallback;
-  flags: Flags;
-  subscribeToFlagChanges: boolean;
-  throwOnInitializationFailure: boolean;
-  flagsUpdateDelayMs?: number;
-}): Promise<any> => {
+const isLaunchDarklyAdapterArgs = (
+  adapterArgs: AdapterArgs
+): adapterArgs is LaunchDarklyAdapterArgs =>
+  (adapterArgs as LaunchDarklyAdapterArgs).clientSideId !== undefined;
+
+const configure = (
+  adapterArgs: AdapterArgs,
+  adapterEventHandlers: AdapterEventHandlers
```

Comment:
Notice how the `configure` function separates the args (provided by the user) from the event handlers